src/hatch_openzim/files_install.py

```diff
@@ -93,9 +98,30 @@ def _process_one_action(
     else:
         raise Exception(f"Unsupported action '{action}'")
 
+    execute_after = action_data.get("execute_after", None)
+    if execute_after:
+        _process_execute_after(base_target_dir=base_target_dir, actions=execute_after)
+
     logger.info("    Done")
 
 
+def _process_execute_after(base_target_dir: Path, actions: List[str]):
+    """execute actions after file(s) installation"""
+
+    for action in actions:
+        logger.info(f"    Executing '{action}'")
+        result = subprocess.run(
+            action,
+            shell=True,  # noqa: S602
+            cwd=base_target_dir,
+            stderr=subprocess.STDOUT,
+            text=True,
+            check=True,
+        )
+        if result:
```

Comment:
This is incorrect. You'll be printing the `CompletedProcess` object.
I'm not sure what you want to achieve here. You're not using `capture_output=True` so the output will printed on stdout (outside of your control). If you want to only print it in your logger command, use `capture_output` and then print `result.stdout`.
By convention, we use `process` or `ps` variable name for those (since it cannot be more explicit given the command is passes)

Reply:
🤦🏻